### ISSUE TITLE
Category settings page gives a 500 error

### ISSUE THREAD

**Issue by s22-tech** [craigk5n/webcalendar#426]
```
PHP Fatal error:  Uncaught TypeError: fclose(): Argument #1 ($stream) must be of type resource, bool given in /home/username/webcal/category.php:17
Stack trace:
#0 /home/username/webcal/category.php(17): fclose(false)
#1 {main}
  thrown in /home/username/webcal/category.php on line 17
```

**Comment by craigk5n** [comment]
Should be fixed in master branch and will be included in 1.9.11 release.
You need to change the permissions of your wc-icons directory to writable to everyone.
```
chmod 777 wc-icons
```

**Comment by s22-tech** [comment]
Actually, permissions of 755 work just fine on linux.

Also, there was no wc-icons directory.  I had to create it by hand.
